src/page-modules/contact/ticket-control-and-fee/formMachine.ts

```diff
@@ -0,0 +1,142 @@
+import { assign, setup } from 'xstate';
+
+export const formMachine = setup({
+  types: {
+    context: {} as { title: string },
+  },
+  actions: {
+    setTitle: assign({
+      title: ({ event }) => event.title,
+    }),
+  },
+}).createMachine({
+  /** @xstate-layout N4IgpgJg5mDOIC5SzAGzAYwC4DED2ATgLYB0sWAhgVgMQYAWeeKJAZmGAMJ5EAOqFAJYA7XISIBtAAwBdRKF7NBWQXmHyQAD0QBaAMwBGAJwkAbAHYArOakGATEamWALEb3OANCACeugy5JLWyMDAz09OztzQzsAX1ivFHRsfGIySmo6RmYwEgIwVgBXYQhUyVkNRVhlVXUkLV1nA2cScya9IwAOAxsOu0s9L18EZz1zEmdO0z1TTudnKT0gsPjEtEwxNPIqWgYmFnZIACMKDABrMuk5eqqatQ1tBB1LA07WqUj-IysDKSM7IaICxvTrWTrmaZOfpSZyrEBJDZlEgUKD5MBEMCiAxZfa5fJFEqXCo3JQqe71R46AymOwkfwuKxdUzWcx2Tw+RBNEzOFx2GYDKTmToOUxwhEpcTI1EcDFYnE5NgcCAnc5E64KUm1B6IKSAhBSMXrCVpFFo2VYAwkQQQdAAQWl6MxFpopo4BiulU15NAj2aUjpL2cUTZ3X55j1+g6gXsTk6EQWplmooS8KNm1IrsdWKlZqddhoEEEsEz7uJGuqZLqPsQYRIbksRlMoSirjjHwj1NpwqMjeFFh5oVhKfF6ZzMqdlsz5tp1rtDunLoddg9JIrWopnPBJE6nUclhpv0bUimEbZlgmdl+ESkx5mi2Ta2So6nE7HWawtMO3D4AhE6YLRaZsuZYgLclbaggMymBMow0tM4RtE4HYBMs-zRHYNKgrMhpPkiL7ZvhH6KlwPD8EIohIio5xgFgADKWCECiYAALJ4BAuSzmAAAqgjUXRDH5DArHsTQI7IrwvAruWdxVg0CDmP4gSNqYPb2Cy3ynnSPzMk4+40noTg4YikqEZO855sR35kX+lG8WcNH0YxQlsRxNrcXZDkCWAzkiWJWAEBQABuaAYFQEBSaBXqyb6th6GYCE7kYTTmDY7LDFSozbil1iqXobY9EZxoZuZBElURX6kb+FGSlR9n8U5LEueJvCiWmJD+UFIVhRFYHrtWCChE4JCREEhj-KCoLIWY4JRD0QTOKyKmFc+ZVmbmoifhwVlVaOtWeQ1wm5B1wWoKFBAQK1T7NT1UUQWECx0hY1KWJYGGxlNsysgpVgwotRjLUi+LFKU4jygcW2VeR6Y3Wu3pyfSrTNDSAymDCeWWHqPJvNeLydDerwKQYAOSkDhKg3sCqHMqpwXOIMMyRB-RxfM7iTMY+5zJMeroTBziJlIswRHG5jE2kVMqrTxBg7kFU-lDaqerD0WcqErT7peix-J91KY5Y2N2DCUx85E2Wi6Q4s02U0t5AUwMK6uDMbk8-RvHyN7HsyYRoXq3T+v4NiNgOv3-XCwgufA9QjmUiuO-1zyJu8nwNj8fwAhyTx65aYxWAbTRRLuQ6PsZWwZFgMfgU7Oj9OMUxsjMoRfHyEYdG8Lj7uYbg3o2ISF6muEmat5d9XJOgpRMQS7qYfMfHMbQRq4cXglYLKjAMnRm2+5qWpx9rrRaQ9w5SDjj8egeo3Ys9pY0ix1jYKNRN8zJxMOaZ4atm95gfysIBfcUNnlXQGxMzGPPAWJBs4vX9tCYUlgN6mQ-htK0bld7jg2l-CC4RLT-13KCf4AwQHp30IpV4bh3CslejCXuUcB57zWqg8qEM5Y2XEOgp2TYTANkTKpauHdwyEMMCCbKPIFjNCFLAl+-cTTv0IptEiTDqppD2vVAKPkwCsP6omcYnCVIhB4RpQhfIs7ZWFOhE2go4HSLKrI7a8saoeWUUxQ6SD0A8T4o5QSjV2LqLkkKEwzIFqhgNjYVkHZMpLxpDeGk3x9wWNoQghhcjrIKNIEo9xjimoUAkt430bIuzuCWEjZoyU9RhFpDeDCoxbBWFcOIouRV4l0PfNYyGzDFH2LSao9qAUTpnQgNkmskR-T-A+AeKYHcaShMXilCJAsHBWAfH3YupBSYg2IP052eVWhBg7oM8IzIrB6hZHSf4Bl+guF3ApDeKykScQAEq2zJmsh2Fc47GH9G0B+uywwY3TomFoKlIhPzZM0KYG8LaqhYc84elItwYWiD0EISxgh6hOa0DWFhHAGyMHrMFSoJY3LcjgPFltIXSReSPKMcLDAdzCKhEpthLSfE1tYe6rJ4jxCAA */
+  id: 'selectForm',
+  initial: 'start',
+  context: {
+    title: '',
+  },
+  states: {
+    start: {
+      on: {
+        'choose.feeComplaintForm': {
+          target: 'agreement1',
+          actions: 'setTitle',
+        },
+        'choose.refundForm': {
+          target: 'refundForm',
+          actions: 'setTitle',
+        },
+        'choose.feedbackForm': {
+          target: 'feedbackForm',
+          actions: 'setTitle',
+        },
+      },
+    },
+
+    agreement1: {
+      initial: 'idleAgreement1',
```

Comment:
Why are these prefixed with `idle`?

Reply:
They are prefixed to idle because initially the user have neither agreed or disagreed with the first agreement. After agreeing, the user can only disagree.